api-2.7/pom.xml

```
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>initializer</artifactId>
+        <groupId>org.openmrs.module</groupId>
```

nit: When referencing external modules, it should be Group-Artifact-Version.

```suggestion
        <groupId>org.openmrs.module</groupId>
        <artifactId>initializer</artifactId>
```